packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -336,7 +336,7 @@ export class FluidDevtools implements IFluidDevtools {
 	 * Gets the registered Container Devtools associated with the provided {@link ContainerKey}, if one exists.
 	 * Otherwise returns `undefined`.
 	 */
-	public getContainerDevtools(containerKey: ContainerKey): IContainerDevtools | undefined {
+	public getContainerDevtools(containerKey: ContainerKey): ContainerDevtools | undefined {
```

Comment:
Two of the big changes that this PR made:
- Removes `readonly` property of `ContainerDevtools.containerData`.
- Changes return type to `ContainerDevtools` so that the instance can access the `containerData`. 

Would be curious to find out more type-safe way (instead of exposing the `ContainerDevtools` type).

Reply:
How about adding a method `IContainerDevtools.addContainerData(data)`? It feels like a better fit because it's already tied to a specific container. Consumers would first call this on the "global devtools object" to get the "devtools object for a specific container", and then interact with that.